.weeder.yaml

```diff
@@ -43,11 +43,17 @@
           - </>
           - expectError
           - expectSuccess
+          - expectFieldEqual
+          - expectFieldNotEqual
+          - getFromResponse
       - module:
         - name: Test.Integration.Framework.Request
         - identifier:
           - ClientError
           - DecodeFailure
+          - Empty
+          - None
+          - NonJson
```

Comment:
These shouldn't be needed anymore I believe :thinking: ?

Reply:
Weeder complains about them when removed...